.github/workflows/lint-commit-message.yml

```diff
@@ -1,13 +1,13 @@
 name: lint-commit-message
-on: [ pull_request ]
+on: [ push,pull_request ]
 
 jobs:
   lint-commit-message:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: 0
       - uses: wagoid/commitlint-github-action@v4
         with:
-          configFile: ${{ github.workspace }}/build/ci/commitlint.config.js
+          configFile: ${{ github.workspace }}/build/ci/commitlint.config.js
```

Comment:
There is a missing "end of file" at the end of the line. Maybe use an IDE plugin to solve this issue once and for all.

Reply:
Got it, I will fix this issue